From b4f8b90fdb99bc5e936193051ff8371b173cd779 Mon Sep 17 00:00:00 2001
From: Zhou Peng-B04994 <eagle.zhou@nxp.com>
Date: Thu, 22 Jun 2017 14:24:12 +0800
Subject: [PATCH 1990/5242] MLK-15132-4 : Enable Hantro decoder on i.MX8MQ

commit  52a58d9bde477d0ad1f41ce64cb778dae4bb708d from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix section mismatch link warning,
Removing unnecessary annotation '__init'/'__exit'

Signed-off-by: Zhou Peng-B04994 <eagle.zhou@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/hantro/hantrodec.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mxc/hantro/hantrodec.c b/drivers/mxc/hantro/hantrodec.c
index 78c2671..3307abb 100755
--- a/drivers/mxc/hantro/hantrodec.c
+++ b/drivers/mxc/hantro/hantrodec.c
@@ -1266,7 +1266,7 @@ void hantrodec_disable_clk(unsigned long value)
  Return type     : int
 ------------------------------------------------------------------------------*/
 
-int __init hantrodec_init(void) {
+int hantrodec_init(void) {
   int result, i;
 
   PDEBUG("module init\n");
@@ -1557,7 +1557,7 @@ int __init hantrodec_init(void) {
  Return type     : int
 ------------------------------------------------------------------------------*/
 
-void __exit hantrodec_cleanup(void) {
+void hantrodec_cleanup(void) {
   hantrodec_t *dev = &hantrodec_data;
   int n =0;
   /* reset hardware */
-- 
1.7.9.5

